File: KendoPractice/src/main/java/com/tatvacoconet/service/IUserServiceUrvashi.java
package com.tatvacoconet.service;

import java.util.List;

import com.tatvacoconet.entity.UserUrvashi;

/**
 * 
 * @author TatvaSoft
 *
 */
public interface IUserServiceUrvashi {
	
	public void save(UserUrvashi user);

	public List<UserUrvashi> listUser();
	
	public void deleteUser(int id);
	
	public void	updateUser(UserUrvashi entity);
	
	public UserUrvashi getUserById(int id); 
		
}

File: KendoPractice/src/main/webapp/resources/app/Demo/Services/IUserServiceUrvashi.ts
/// <reference path='../../_allUrvashi.ts' />
module DemoUrvashi {
    export interface IUserServiceUrvashi {
    	saveUser : (user: UserUrvashi)  => ng.IPromise<string>;
    	listUser : ()  => ng.IPromise<Array<UserUrvashi>>;
    	deleteUser : (id:any) => ng.IPromise<string>;
    	updateUser : (user:UserUrvashi)  => ng.IPromise<string>;
    	getUserById : (id:number)=>ng.IPromise<UserUrvashi>;
    }
}